lib/ruby2d/window.rb

```diff
@@ -236,6 +223,36 @@ def add_object(o)
         false
       end
     end
-    
+
+    def events_store(event_type)
```

Comment:
Here's another idea: now that all the event "stores" are hashes, why don't we just make an `events` hash to put all these in.